src/parser/jobs/drg/modules/BloodOfTheDragon.tsx

```diff
@@ -245,17 +268,14 @@ export default class BloodOfTheDragon extends Analyser {
 		}
 	}
 
-	onDeath() {
+	onDeathLotd() {
 		// RIP
 		this.updateGauge()
 		this.lifeDuration = 0
 		this.finishLifeWindow()
-
-		// TODO: check if you lose eyes on death
-		this.eyes = 0
 	}
 
-	onRaise(event: Event) {
+	onRaiseLotd(event: Event) {
 		// So floor time doesn't count against BotD uptime
 		this.lastEventTime = event.timestamp
 	}
```

Comment:
Is this even still needed?  Since BotD is a passive now, there's no uptime tracking to maintain for that, correct?

Reply:
yep looks like it's unnecessary now, let me get that outta here